pkg/services/ngalert/api/lotex_ruler.go

```diff
@@ -203,10 +204,14 @@ func (r *LotexRuler) validateAndGetPrefix(ctx *models.ReqContext) (string, error
 	subtype := ctx.Query(subtypeQuery)
 	subTypePrefix, ok := subtypeToPrefix[subtype]
 	if !ok {
-		r.log.Debug("unable to determine prometheus datasource subtype, using default prefix", "subtype", subtype)
+		r.log.Debug(
+			"unable to determine prometheus datasource subtype, using default prefix",
+			"datasource", ds.Uid, "datasource_type", ds.Type, "subtype", subtype, "prefix", prefix)
 		return prefix, nil
 	}
 
+	r.log.Debug("determined prometheus datasource subtype",
+		"datasource", ds.Uid, "datasource_type", ds.Type, "subtype", subtype)
```

Comment:
Grafana logging guidelines prefer a camelCase name for log keys: `datasourceType`. If this is the first occurrence of this key, let's camelCase it. If there are other usages, let's be consistent with that.